crates/ty_python_semantic/resources/mdtest/annotations/annotated.md

```diff
@@ -76,7 +76,13 @@ from ty_extensions import reveal_mro
 
 class C(Annotated[int, "foo"]): ...
 
-reveal_mro(C)  # revealed: (<class 'C'>, <class 'int'>, <class 'object'>)
+# revealed: (<class 'C'>, <class 'int'>, <class 'object'>)
+reveal_mro(C)
+
+class D(Annotated[list[str], "foo"]): ...
+
+# revealed: (<class 'D'>, <class 'list[str]'>, <class 'MutableSequence[str]'>, <class 'Sequence[str]'>, <class 'Reversible[str]'>, <class 'Collection[str]'>, <class 'Iterable[str]'>, <class 'Container[str]'>, typing.Protocol, typing.Generic, <class 'object'>)
+reveal_mro(D)
```

Comment:
Is it worth also adding a test for a variant of this:

```py
class E(Annotated[list["E"], "metadata"]): ...
```

(which works fine at runtime, I think)